### ISSUE TITLE
Provided username and password not working

### ISSUE THREAD

**Issue by MeSlavFisherman** [D10S0VSkY-OSS/Stack-Lifecycle-Deployment#45]
Initial password or username not working (wrong user or password)
![image](https://user-images.githubusercontent.com/86969404/144863476-675fde29-7e52-4769-a305-988b6542ec88.png)

**Comment by MeSlavFisherman** [comment]
I am providing additional screenshots
![image](https://user-images.githubusercontent.com/86969404/144864630-08a1d62c-fad4-4fe1-b7f6-8f09af9941ef.png)

![image](https://user-images.githubusercontent.com/86969404/144864674-6d1fec37-0576-48a9-8d60-5fa6d0c38b73.png)

**Comment by D10S0VSkY-OSS** [comment]
Heelo @MeSlavFisherman  
Please can you tell me what version of sld you are testing?

**Comment by MeSlavFisherman** [comment]
I used the latest version on master branch (2.0.1)

**Comment by D10S0VSkY-OSS** [comment]
I am not able to reproduce your problem. I can log in correctly
Can you send me logs for pods and outputs pods state?
have you tried to access from the api?
**http://0.0.0.0:8000/docs**
or via curl
```
curl -X 'POST' \
  'http://0.0.0.0:8000/api/v1/authenticate/access-token-json' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{
  "username": "admin",
  "password": "Password08@"
}'
```

**Comment by MeSlavFisherman** [comment]
![image](https://user-images.githubusercontent.com/86969404/145029582-1c13b6c7-1d9b-43c5-98f2-a2dba5e6b3f1.png)

![image](https://user-images.githubusercontent.com/86969404/145028992-e20f22af-e34a-4c05-8aaf-8811a50cc461.png)

![image](https://user-images.githubusercontent.com/86969404/145029100-cc380e1e-f431-4f4e-8df5-b70951200bed.png)

![image](https://user-images.githubusercontent.com/86969404/145029342-9d9016f3-3489-4cc9-bb9c-af7d0d9a68dd.png)

![image](https://user-images.githubusercontent.com/86969404/145029417-b7952b21-a888-4808-b885-cf4ddd25fbd9.png)

![image](https://user-images.githubusercontent.com/86969404/145029736-d6434bac-03dc-4dc4-956a-4e79a6c4c72a.png)

![image](https://user-images.githubusercontent.com/86969404/145029817-ac9fa27d-1ceb-4637-8619-249c1cb2e4e6.png)

![image](https://user-images.githubusercontent.com/86969404/145029932-2a7501e1-aa3d-4256-b620-aef19420259f.png)

![image](https://user-images.githubusercontent.com/86969404/145030005-982ed756-38da-4417-b54f-7266391b42ae.png)

**Comment by MeSlavFisherman** [comment]
I found the bug, my SLD was up on localhost:5000, but in kplay.sh it was using 0.0.0.0 which apparently is not the same. I changed kplay.sh script to POST it on localhost instead of 0.0.0.0

**Comment by D10S0VSkY-OSS** [comment]
Hi @MeSlavFisherman ,
Thanks for the feedback.
fix is ​​created in release v2.1.1 #46 
https://github.com/D10S0VSkY-OSS/Stack-Lifecycle-Deployment/commit/40a81147f887b5ec3a3577bd3502010b6270bca6
